### ISSUE TITLE
Unable to open /requirements.yml

### ISSUE THREAD

**Issue by gabrielgiordan** [roots/trellis#697]
- [x] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [ ] This is a feature request
- [x] This is a bug report
- [x] This request isn't a duplicate of an [existing issue](https://github.com/roots/trellis/issues)
- [x] I've read the [docs](https://roots.io/trellis/docs) and followed them (if applicable)

---

**What is the current behavior?**

I cannot vagrant provision the latest Trellis (master) on Windows 10.

**What is the expected or desired behavior?**

Be able to provision

---

## Bug report

**Please provide steps to reproduce, including full log output:**

I can ssh into it, but when vagrant up or vagrant provision it outputs this log:

`vagrant provision`

```
==> default: Running Ansible Galaxy install
==> default:  [ERROR]: Unable to open /requirements.yml: [Errno 2] No such file or
==> default: directory: '/requirements.yml'
==> default: Running Ansible Playbooks
==> default: ERROR! the playbook: dev.yml could not be found
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what went wrong.
```

`vagrant provision --debug`

```
 INFO interface: info: ==> default: You are using pip version 8.1.1, however version 9.0.1 is available.
==> default: You should consider upgrading via the 'pip install --upgrade pip' command.
==> default: You are using pip version 8.1.1, however version 9.0.1 is available.
==> default: You should consider upgrading via the 'pip install --upgrade pip' command.
DEBUG ssh: Sending SSH keep-alive...
 INFO interface: info: Requirement already satisfied (use --upgrade to upgrade): markupsafe in /usr/local/lib/python2.7/dist-packages
 INFO interface: info: ==> default: Requirement already satisfied (use --upgrade to upgrade): markupsafe in /usr/local/lib/python2.7/dist-packages
==> default: Requirement already satisfied (use --upgrade to upgrade): markupsafe in /usr/local/lib/python2.7/dist-packages
DEBUG ssh: stderr: You are using pip version 8.1.1, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

 INFO interface: info: You are using pip version 8.1.1, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
 INFO interface: info: ==> default: You are using pip version 8.1.1, however version 9.0.1 is available.
==> default: You should consider upgrading via the 'pip install --upgrade pip' command.
==> default: You are using pip version 8.1.1, however version 9.0.1 is available.
==> default: You should consider upgrading via the 'pip install --upgrade pip' command.
 INFO interface: info: Running Ansible Galaxy install
 INFO interface: info: ==> default: Running Ansible Galaxy install
==> default: Running Ansible Galaxy install
DEBUG ssh: stderr:  [ERROR]: Unable to open /requirements.yml: [Errno 2] No such file or
directory: '/requirements.yml'

 INFO interface: info:  [ERROR]: Unable to open /requirements.yml: [Errno 2] No such file or
directory: '/requirements.yml'
 INFO interface: info: ==> default:  [ERROR]: Unable to open /requirements.yml: [Errno 2] No such file or
==> default: directory: '/requirements.yml'
==> default:  [ERROR]: Unable to open /requirements.yml: [Errno 2] No such file or
==> default: directory: '/requirements.yml'
 INFO interface: info: Running Ansible Playbooks
 INFO interface: info: ==> default: Running Ansible Playbooks
==> default: Running Ansible Playbooks
DEBUG ssh: stderr: ERROR! the playbook: dev.yml could not be found

 INFO interface: info: ERROR! the playbook: dev.yml could not be found
 INFO interface: info: ==> default: ERROR! the playbook: dev.yml could not be found
==> default: ERROR! the playbook: dev.yml could not be found
DEBUG ssh: Exit status: 1
ERROR warden: Error occurred: The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what went wrong.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what went wrong.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what went wrong.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what went wrong.
 INFO warden: Beginning recovery process...
 INFO warden: Calling recover: #<Vagrant::Action::Builtin::Call:0x6039eb8>
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
 INFO warden: Recovery complete.
ERROR warden: Error occurred: The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what went wrong.
 INFO warden: Beginning recovery process...
 INFO warden: Recovery complete.
ERROR warden: Error occurred: The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
```

If I ssh into the box the the requirements.yml file is at the /vagrant folder

```
vagrant@amilempresarialonline:/vagrant$ ls
ansible.cfg   dev.yml     README.md         Vagrantfile
bin           group_vars  requirements.yml  variable-check.yml
CHANGELOG.md  hosts       roles             xdebug-tunnel.yml
deploy-hooks  lib         rollback.yml
deploy.yml    LICENSE.md  server.yml
```

**Please describe your local environment:**

Ansible version: 2.0.2 and 2.2.0

OS: Windows 10

Vagrant version: 1.9.0

**Where did the bug happen? Development or remote servers?**

Development

---

**Comment by fullyint** [comment]
fixed in #696
but let us know if that didn't resolve it for you

**Comment by gabrielgiordan** [comment]
Hey many thanks @fullyint I think this issue is fixed now! 👍 

But I'm having a problem with another issue on my encrypted vaults, it's related to this [post](https://discourse.roots.io/t/shared-trellis-folder-permissions-vault-problem/6640/2). In the past, the only thing that worked is the the [@aoe answer](https://discourse.roots.io/t/shared-trellis-folder-permissions-vault-problem/6640/7?u=gabrielgiordan)

I've tried almost anything, using ask_vault_pass, changing chmod from bash, changing permission on file properties... but unfortunately I didn't find any solution to remove the executable bit from the .vault_pass
